kubernetes/deployment-staging.tmpl

```diff
@@ -115,6 +120,13 @@ spec:
               mountPath: "/static-assets"
             - name: caesar-nginx-config
               mountPath: "/etc/nginx-sites"
+          readinessProbe:
+            httpGet:
+              path: /
+              port: 80
+              initialDelaySeconds: 5
```

Comment:
Will this be long enough?

Reply:
probably not tbh and it really depends how long migrations take to run....? It might be worthwhile thinking about this broadly and how we run migrations in caesar (container boot) as well.

I can get some realistic boot times under normal (no / quick migrations) etc to better estimate a value.